.github/workflows/build-citus-community-nightlies.yml

```
@@ -17,6 +17,11 @@ on:
     - cron: "30 1 * * *"
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

How about using a simpler expression that only contains workflow and git reference name?

```suggestion
  group: ${{ github.workflow }}-${{ github.ref }}
```

This is the suggested approach in the documentation [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow).

---

Note: This comment is valid for all the concurrency.group attributes introduced in this PR.

---

Changed for all pipelines